### PR TITLE
Update Inflation Page

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-inflation.md [hunk 1]
```diff
@@ -11,19 +11,20 @@ import RPC from "./../../components/RPC-Connection";
 
 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} is an inflationary token. On the
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network, inflation is
-{{ polkadot: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/polkadot/src/lib.rs#L576), :polkadot }}
-{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535), :kusama }}
-which is distributed as staking rewards based on the amount staked vs. ideal staking rate, and the
-remainder goes to the treasury.
+{{ polkadot: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/polkadot/src/lib.rs#L576). :polkadot }}
+{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535). :kusama }}Depending
+on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply locked and the ideal staking
```
Comment (profradha) [inline docs/learn/learn-inflation.md]:
```suggestion
on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply staked and the ideal staking
```

File: docs/learn/learn-inflation.md [hunk 2]
```diff
@@ -11,19 +11,20 @@ import RPC from "./../../components/RPC-Connection";
...
+rate (more about this below), part of the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
+inflation goes to stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).
```
Comment (profradha) [inline docs/learn/learn-inflation.md]:
```suggestion
inflation is distributed to the stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).
```

File: docs/learn/learn-inflation.md [hunk 3]
```diff
@@ -11,19 +11,20 @@ import RPC from "./../../components/RPC-Connection";
...
 
 :::info
 
 DOT went through [redenomination](./learn-redenomination.md) in 2020 that saw the DOT token supply
 increase by 100 times.
 
 The current token supply on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is
-{{ polkadot: <RPC network="polkadot" path="query.balances.totalIssuance" defaultValue={12230666300429914781} filter="humanReadable"/> (Over 1.2 Billion DOT). :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093} filter="humanReadable"/> (Over 12 Million KSM). :kusama }}
```
Comment (profradha) [inline docs/learn/learn-inflation.md:26]: image

Knowing the quantity of DOT or KSM by taking a glance at this number is tough (especially for those who are not from the ecosystem and who do not have any idea about the total issuance. That is why an approximate human readable number quantifying the supply was added. If you plan on making this accessible for diverse audience, either automate this in the human readable filter or keep the approximate values as is.

File: docs/learn/learn-inflation.md [hunk 4]
```diff
@@ -35,46 +36,49 @@ network through funding the on-chain treasury. There is an opportunity cost of k
...
 The chart below shows the inflation model of the network. Depending on the number of staked tokens,
 the distribution of the inflation to validators and nominators versus the treasury will change
 dynamically to provide incentives to participate (or not participate) in staking.
 
-There is a [dynamic _ideal staking rate_](#ideal-staking-rate) (in the figure set to 0.5 or 50%)
-that the network tries to maintain. The goal is to have the _system staking rate_ meet the _ideal
-staking rate_. The system staking rate would be the total amount staked over the total token supply,
+There is a [dynamic _ideal staking rate_](#ideal-staking-rate) (in the figure set to 0.6 or 60%)
+that the network tries to maintain. The inflation model will incentivize network participants to
+stake when the _system staking rate_ < _ideal staking rate_ and disincentivize staking when _system
```
Comment (profradha) [inline docs/learn/learn-inflation.md]: At first glance, the word "system staking rate" is ambiguous to a new reader, making a reader think it is somehow dependent on the system or protocol. An apt name would be "current staking rate" or "staking rate in the current era" etc. If we already agreed on standardizing "system staking rate" in earlier discussions that I'm not aware of, you can ignore this suggestion.

```suggestion
stake when the _current staking rate_ < _ideal staking rate_ and disincentivize staking when _current
```

File: docs/learn/learn-inflation.md [hunk 5]
```diff
@@ -35,46 +36,49 @@ network through funding the on-chain treasury. There is an opportunity cost of k
...
+staking rate_ > _ideal staking rate_. The goal is to have the system staking rate meet the ideal
```
Comment (profradha) [inline docs/learn/learn-inflation.md]:
```suggestion
staking rate_ > _ideal staking rate_. The goal is to have the staking rate meet the ideal
```

File: docs/learn/learn-inflation.md [hunk 6]
```diff
@@ -35,46 +36,49 @@ network through funding the on-chain treasury. There is an opportunity cost of k
...
+staking rate. The system staking rate would be the total amount staked over the total token supply,
 where the total amount staked is the stake of all validators and nominators on the network. The
```
Comment (profradha) [inline docs/learn/learn-inflation.md]: Better to move this definition earlier in this section, before introducing the term.

```suggestion
staking rate. The current staking rate would be the total amount staked in the current era over the total token supply,
where the total amount staked is the stake of all validators and nominators on the network. The
```

**Review by DrW3RK — COMMENTED** [review]
Looks good. Thanks for updating. Also, Could we use Polkadot Pink color instead of Red in the figure? :) I don't want readers to perceive inflation as some red flag

File: docs/learn/learn-inflation.md (hunk 3 above)
Comment (filippoweb3) [reply docs/learn/learn-inflation.md:26]: re-added, will create a filter in another PR

**Review by DrW3RK — APPROVED** [review]
Approved from my end.

**Pull request merged**